.eslintrc.js

```diff
@@ -47,7 +51,36 @@ module.exports = {
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
```

Comment:
```suggestion
      extends: ['plugin:node/recommended'],
      rules: {
```

Reply:
This appears to have been doing this because [using `extends` in `overrides` is unsupported](https://github.com/eslint/eslint/issues/8813).